projects/vdk-core/src/vdk/internal/builtin_plugins/connection/impl/router.py

```diff
@@ -52,7 +52,7 @@ def add_open_connection_factory_method(
         """
         Add new connection factory method. See parent doc for more.
         """
-        self._connection_builders[dbtype] = open_connection_func
+        self._connection_builders[dbtype.lower()] = open_connection_func
```

Comment:
ingest method is case insenstive  
data job name is case insenstive (well it's always lower-case)

team name is case sensitive. Though I tihnk it should not be. 


Unless there's very good reason (like People's names) where the capital letter has some actual semantic meaning. we should design for case-insensitive.

Reply:
maybe add that info to `api/plugin/plugin_input.py` in docs of `IManagedConnectionRegistry add_open_connection_factory_method()`?

and since api-documented, should we add a unit test

Reply:
I will update the doc of add_open_connection_factory_method

I added a unit test in `test_connection_router` .